client/a8c-for-agencies/sections/purchases/billing/billing-details/style.scss

```diff
@@ -0,0 +1,106 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.billing-details {
+	&__row {
```

Comment:
Can we append all `&__` with `.billing-details`? This will help us find the styles easily.